lib/spark_api/models/listing.rb

```diff
@@ -235,6 +235,10 @@ def respond_to?(method_symbol, include_all=false)
         end
       end
 
+      def batch_photo_delete(photoIds, args={})
+        connection.delete "#{self.class.path}/#{self.Id}/photos/#{photoIds}", args
```

Comment:
This is the second reference to `"#{self.class.path}/#{self.Id}/photos/"` in this class. It seems helpful to extract out a private `subresource_path(subresource)` method or a `photos_subresource_path()` method to clarify and DRY things out a bit.